pastas_plugins/modflow/modflow.py

```diff
@@ -83,12 +78,18 @@ def create_model(self) -> None:
             ncol=1,
             delr=1,
             delc=1,
-            top=1000,
-            botm=-1000,
+            top=1.0,
+            botm=0.0,
```

Comment:
i don't know if this works with constant_d..

Reply:
The way it's set up now modflow doesn't do anything with the constant. It just computes stuff relative to 0.0.

Reply:
And the model is confined now.